lib/sprangular/engine.rb

```diff
@@ -13,6 +13,12 @@ class Engine < ::Rails::Engine
       Rails.application.config.assets.precompile += %w( images/* bootstrap/* )
     end
 
+    initializer "sprangular.api_domain", :before => :load_config_initializers do |app|
```

Comment:
Unused block argument - `app`. You can omit the argument if you don't care about it.<br>Line is too long. [86/80]<br>Use the new Ruby 1.9 hash syntax.